src/main/java/com/ticketbookingapp/domain/booking/Reservation.java

```diff
@@ -0,0 +1,43 @@
+package com.ticketbookingapp.domain.booking;
+
+
+import com.ticketbookingapp.domain.seat.Seat;
+import com.ticketbookingapp.domain.ticket.Ticket;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
```

Comment:
Domena czy jednak warstwa zapisu do bazy danych ;) ?

Ładnie podzieliłeś projekt na warstwy i widzę, że próbowałeś napisać to w stylu hexagonal-architecture - to plus :) W tym podejściu warstwa domenowa powinna być jednak odseparowana od zapisu do bazy danych.

Reply:
Nie ukrywam, że styl hexagonal-architecture to dla mnie coś nowego, co teraz spróbowałem zrobić i jeszcze nie wszystko jest dla mnie w nim oczywiste. W tym przypadku nie zastosowałem odseparowania w ramach uproszczenia, które poparte jest w materiale https://www.youtube.com/watch?v=JJXmgCx_wh0 (Podejście 4)